opentracing/tracecontext.go

```diff
@@ -49,18 +49,20 @@ type TraceContext interface {
 	// HTTP headers are case insensitive.
 	//
 	// `restrictedKey` MUST match the regular expression
-	// `(?i:[a-z0-9][-a-z0-9]*)` and is case-insensitive. See
-	// CanonicalizeTraceTagKey().
+	// `(?i:[a-z0-9][-a-z0-9]*)` and is case-insensitive. That is, it must
```

Comment:
I'd prefer documenting this as down-cased, or that case won't be preserved. case-insensitivity is more about comparison than storage.